specutils/io/default_loaders/tests/test_desi.py

```diff
@@ -0,0 +1,141 @@
+    assert spectrum[1].meta['band'] == 'r'
+    assert spectrum[2].meta['band'] == 'z'
+
+
+@pytest.mark.parametrize('local_filename, fmt', [('coadd-sv3-dark-26065.fits', 'DESI coadd'),
+                                                 ('spectra-sv3-dark-26065.fits', 'DESI spectra'),
+                                                 ('coadd-5-169-thru20210419.fits', 'DESI coadd'),
+                                                 ('spectra-5-169-thru20210419.fits', 'DESI spectra')], indirect=['local_filename'])
+def test_SpectrumList_reader_hdulist(local_filename, fmt):
+    """Test reading a HDUList.
+    """
+    with fits.open(local_filename, mode='readonly') as hdulist:
+        spectrum = SpectrumList.read(hdulist, format=fmt)  # noqa
+    assert spectrum[0].meta['band'] == 'b'
+    assert spectrum[1].meta['band'] == 'r'
+    assert spectrum[2].meta['band'] == 'z'
+
+
+@pytest.mark.parametrize('remote_filename, loader', [pytest.param('coadd-sv3-dark-26065.fits', coadd_loader, marks=pytest.mark.remote_data),
+                                                     pytest.param('spectra-sv3-dark-26065.fits', spectra_loader, marks=pytest.mark.remote_data),
+                                                     pytest.param('coadd-5-169-thru20210419.fits', coadd_loader, marks=pytest.mark.remote_data),
+                                                     pytest.param('spectra-5-169-thru20210419.fits', spectra_loader, marks=pytest.mark.remote_data)], indirect=['remote_filename'])
+def test_loader_remote(tmp_path, remote_filename, loader):
+    """Remote data test of 'bare' loader functions.
+    """
+    with set_temp_cache(path=str(tmp_path)):
+        filename = download_file(remote_filename, cache=True)
+        spectrum = loader(filename)  # noqa
+        assert spectrum[0].meta['band'] == 'b'
+        assert spectrum[1].meta['band'] == 'r'
+        assert spectrum[2].meta['band'] == 'z'
+
+
+@pytest.mark.parametrize('remote_filename, fmt', [pytest.param('coadd-sv3-dark-26065.fits', 'DESI coadd', marks=pytest.mark.remote_data),
+                                                  pytest.param('spectra-sv3-dark-26065.fits', 'DESI spectra', marks=pytest.mark.remote_data),
+                                                  pytest.param('coadd-5-169-thru20210419.fits', 'DESI coadd', marks=pytest.mark.remote_data),
+                                                  pytest.param('spectra-5-169-thru20210419.fits', 'DESI spectra', marks=pytest.mark.remote_data)], indirect=['remote_filename'])
+def test_SpectrumList_reader_remote(tmp_path, remote_filename, fmt):
+    """Remote data test of class method.
+    """
+    with set_temp_cache(path=str(tmp_path)):
+        filename = download_file(remote_filename, cache=True)
+        spectrum = SpectrumList.read(filename, format=fmt)  # noqa
+        assert spectrum[0].meta['band'] == 'b'
+        assert spectrum[1].meta['band'] == 'r'
+        assert spectrum[2].meta['band'] == 'z'
```

Comment:
```suggestion
```

I thought the local file tests were supposed to replace these remote tests - they take a very long time, I don't think we want to keep them.

Reply:
Remote tests have been removed. I added some documentation to the test file on the origin of the local files.